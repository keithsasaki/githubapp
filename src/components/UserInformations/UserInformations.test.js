import React from 'react';
import '@testing-library/jest-dom';

import renderer from 'react-test-renderer';
import UserInformation from '.';

describe("Testing UserInformation Component", () => {
    it("matches snapshot", () => {
        const user = {
            avatar_url:"https://avatars2.githubusercontent.com/u/24439327?s=460&u=7b1ce31ae270195192c94f37116ffa46fcca572d&v=4",
            name: "Keith",
            login: "keithsasaki",
            company: "Concrete",
            location: "SÃ£o Carlos",
            public_repos: 200,
            followers: 1000
        }

        const tree = renderer.create(<UserInformation user={user} starsSum={50}/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
})