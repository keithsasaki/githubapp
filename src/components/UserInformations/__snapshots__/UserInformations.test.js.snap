// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing UserInformation Component matches snapshot 1`] = `
<div
  id="user-informations"
>
  <img
    alt="user avatar"
    id="user-avatar"
    src="https://avatars2.githubusercontent.com/u/24439327?s=460&u=7b1ce31ae270195192c94f37116ffa46fcca572d&v=4"
  />
  <span
    id="user-name"
  >
    Keith
  </span>
  <span
    id="user-login"
  >
    keithsasaki
  </span>
  <div
    id="icon-and-text"
  >
    <div
      className="icon-and-text"
    >
      <img
        alt="Organization Icon"
        src="organization.svg"
      />
      <span>
         
        Concrete
         
      </span>
    </div>
    <div
      className="icon-and-text"
    >
      <img
        alt="Location Icon"
        src="location.svg"
      />
      <span>
         
        SÃ£o Carlos
         
      </span>
    </div>
    <div
      className="icon-and-text"
    >
      <img
        alt="Star Icon"
        src="star.svg"
      />
      <span>
         
        50
         
      </span>
    </div>
    <div
      className="icon-and-text"
    >
      <img
        alt="Repositories Icon"
        src="repositories.svg"
      />
      <span>
         
        200
         
      </span>
    </div>
    <div
      className="icon-and-text"
    >
      <img
        alt="Followers Icon"
        src="followers.svg"
      />
      <span>
         
        1000
         
      </span>
    </div>
  </div>
</div>
`;
